src/test/java/org/jabref/model/groups/AutomaticPersonsGroupTest.java

```diff
@@ -0,0 +1,59 @@
+package org.jabref.model.groups;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.strings.LatexToUnicodeAdapter;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class AutomaticPersonsGroupTest {
+    private static Set<GroupTreeNode> createPersonSubGroupFrom(String... lastNames) {
+        return Arrays.stream(lastNames)
+                     .distinct()
+                     .map(LatexToUnicodeAdapter::format)
+                     .map(lastName ->
+                             new LastNameGroup(lastName, GroupHierarchyType.INDEPENDENT, StandardField.AUTHOR, lastName))
+                     .map(GroupTreeNode::new)
+                     .collect(Collectors.toSet());
+    }
+
+    @Test
+    void createSubgroupsFromCommaSeparatedLastNames() {
+        BibEntry bibEntry = new BibEntry().withField(StandardField.AUTHOR, "Turing, Alan and Hopper, Grace");
+        var subgroups = new AutomaticPersonsGroup("", GroupHierarchyType.INDEPENDENT, StandardField.AUTHOR).createSubgroups(bibEntry);
+        var expectedSubgroups = createPersonSubGroupFrom("Turing", "Hopper");
+        assertEquals(expectedSubgroups, subgroups);
+    }
+
+    @Test
+    void createSubgroupsContainingCommaSeparatedLastNames() {
+        BibEntry bibEntry = new BibEntry().withField(StandardField.AUTHOR, "Turing, Alan and Hopper, Grace");
+        var subgroups = new AutomaticPersonsGroup("", GroupHierarchyType.INDEPENDENT, StandardField.AUTHOR).createSubgroups(bibEntry);
+        var expectedSubgroups = createPersonSubGroupFrom("Turing", "Hopper");
+        assertEquals(expectedSubgroups, subgroups);
+    }
+
+    @Test
+    void createSubgroupFromLatexAndCheckForUnicodeLastName() {
+        BibEntry bibEntry = new BibEntry().withField(StandardField.AUTHOR, "Kurt G{\\\"{o}}del");
+        BibEntry godelEntry = new BibEntry().withField(StandardField.AUTHOR, "Kurt Gödel");
```

Comment:
godelEntry is not used, right? (here and in the following test)

Reply:
Nope. I really need to stop ignoring my IDE warnings, at least in code I have written myself. Sorry about that.